chain/chain/src/store.rs

```diff
@@ -2252,9 +2253,10 @@ impl<'a> ChainStoreUpdate<'a> {
 
     pub fn gc_outgoing_receipts(&mut self, block_hash: &CryptoHash, shard_id: ShardId) {
         let mut store_update = self.store().store_update();
-        match self.get_outgoing_receipts(block_hash, shard_id).map(|receipts| {
-            receipts.iter().map(|receipt| receipt.receipt_id.clone()).collect::<Vec<_>>()
-        }) {
+        match self
+            .get_outgoing_receipts(block_hash, shard_id)
+            .map(|receipts| receipts.iter().map(|receipt| receipt.receipt_id).collect::<Vec<_>>())
```

Comment:
I feel it's better to pull this operation into the `Ok` branch:

```
match  self.get_outgoing_receipts(block_hash, shard_id) {
  Ok(receipts) => {
    let receipt_ids = receipts.iter().map(|receipt| receipt.receipt_id).collect::<Vec<_>>();
  }
}
```

it might even make the collect unnecessary.